roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -6,8 +6,8 @@
   register: result
   until: result is succeeded
   when:
-    - not containerized_deployment
     - ansible_os_family in ['RedHat', 'Suse']
+  tags: package-install
```

Comment:
I like adding this tag because it fixes https://github.com/ceph/ceph-ansible/issues/4197

Reply:
+1